packages/dashmate/test/e2e/testnetEvonode.spec.js

```diff
@@ -207,4 +210,61 @@ describe('Testnet Evonode', function main() {
       await assertServiceRunning(config, 'core', false);
     });
   });
+
+  describe('dashmate helper', () => {
+    it('should be able to request HTTP to helper api', async () => {
+      config.set('dashmate.helper.api.port', 41337);
+      config.set('dashmate.helper.api.enable', true);
+
+      await configFileRepository.write(configFile);
+
+      const writeConfigTemplates = container.resolve('writeConfigTemplates');
+
+      writeConfigTemplates(config);
+
+      const startNodeTask = container.resolve('startNodeTask');
+
+      const task = startNodeTask(config);
```

Comment:
Shall we do it before stop so we don't need to wait until node is started again?